docs/design/node-resource-reservation-design.md

```diff
@@ -0,0 +1,52 @@
+# Volcano node resource reservation
+## background
+* Consider such situation: there are thounsands of pods to be scheduled every day, in 1 to 2 o'clock 500 low priority pods are created and scheduled which used 99% of cluster resource, in 2 to 3 o'clock 10 high priority pods are created, however, low priority pods are still running, high priority pods can not be scheduled due to lack of resource.
+* Users want high priority tasks in 2 to 3 o'clock have resource to schedule immediately every day and high priority tasks not preempt low priority pods because some low priority pods have already run many days.
+## design
+![annotation](images/node-resource-reservation-annotation.png) 
+### recognize high priority pods
```

Comment:
It seems that it should not be called "high priority" but "need to be reserved" podgroup. High priority seems to have the priority field definition. 🤔

Reply:
updated